pcsx2/gui/AppMain.cpp

```diff
@@ -534,11 +535,7 @@ void DoFmvSwitch(bool on)
 				viewport->DoResize();
 	}
 
-	if (EmuConfig.Gamefixes.FMVinSoftwareHack) {
```

Comment:
Isn’t this if missing now?

Reply:
Oh yeah, that should still be there. I'll fix it in a few....